prql-compiler/tests/integration/docker-compose.yml

```diff
@@ -1,23 +1,46 @@
-version: "3"
 services:
   postgres:
-    build:
-      dockerfile: postgres.dockerfile
-      context: .
-    healthcheck:
-      test: /usr/bin/pg_isready -U postgres
-      interval: 5s
-      timeout: 10s
-      retries: 120
+    image: "postgres:alpine"
     ports:
-      - 5454:5432
-  integration:
-    build:
-      dockerfile: integration.dockerfile
-      context: .
+      - "5432:5432"
     environment:
-      POSTGRES_HOST: "postgres"
-      RUST_BACKTRACE: "0"
-    depends_on:
-      postgres:
-        condition: service_healthy
+      POSTGRES_DB: dummy
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: root
+    volumes: &vol
+      - ./data/chinook:/tmp/chinook:ro
+  mysql:
+    # No arm64 image available; remove when one does become available
+    platform: linux/amd64
+    image: "mysql:debian"
```

Comment:
Just a question. Why did you choose `debian` instead of `oracle` in the original PR (#2286)?

`oracle` seems to have a history of being added for arm64 support. (docker-library/mysql#680)

```sh
$ docker buildx imagetools inspect mysql:debian
Name:      docker.io/library/mysql:debian
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:01003f288c58133dad270905e2768e6a1d399d709a4a1375bc4f982f747252c3

Manifests:
  Name:      docker.io/library/mysql:debian@sha256:38d6c177f28baa3b93350a59f0aae2c032418f05e2efd979c9da22d0a27d6223
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64
```

```sh
$ docker buildx imagetools inspect mysql:oracle
Name:      docker.io/library/mysql:oracle
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:ff3828b105886f496e47eddced1676b2abd1419a40a65ac4ab387ba1c538a09f

Manifests:
  Name:      docker.io/library/mysql:oracle@sha256:60eccd02d676ed2214c07be7292da931717f89efffafc2d1f66456daee7c4dc6
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64

  Name:      docker.io/library/mysql:oracle@sha256:2cb5e59c2166e94a3fb7e4da239e09112097af483f9f2bc8f1f87e3394fdf533
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64/v8
```

Reply:
No specific reason. tbh I copied the section with mysql from the internet :wink:. And that includes the comment

Reply:
I see.
Perhaps there is worth changing to `oracle` here.